src/Paprika/RLP/Rlp.cs

```diff
@@ -0,0 +1,65 @@
+using Nethermind.Int256;
+using Paprika.Utils;
+
+namespace Paprika.RLP;
+
+public static class Rlp
```

Comment:
There are multiple "magic numbers" in the class. We should extract them into constants and document the reason for their existence (spec, optimization, etc.)